docs/topic-guides/training-units.txt

```diff
@@ -0,0 +1,180 @@
+.. _topic-guides_training-units:
+
+################
+ Training Units
+################
+
+With the flexibility of Determined’s experiment configuration, the combinations of possible
```

Comment:
I would introduce this topic differently: we should be explaining why the reader might want to understand the information that follows. Perhaps something like:

```
When training a single model or performing a hyperparameter search,
it can be useful to specify how much data a model should be trained
on before an action is taken (e.g., before training is terminated or
before a checkpoint of the state of the model is taken). Determined
supports a flexible system of _training units_ to specify this. etc etc.
```

Reply:
This is much better than my intro, thanks! I'll keep explaining why they might be interested in mind for intros.